.github/workflows/ci.yml

```
@@ -0,0 +1,39 @@
+
+on:
+  pull_request:
+    branches:
+      - mc-develop
+  push:
+    branches:
+      - mc-develop
+
+env:
+  RUST_BACKTRACE: full
+  CARGO_INCREMENTAL: 0
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Should these be using the self hosted runners?
```suggestion
    runs-on: [self-hosted, Linux, large]
    container: mobilecoin/rust-sgx-base:latest
```

---

I don't think it's necessary, TBH.

---

The main benefit is this
> To reduce our spend we should go through and start transitioning GHA jobs that use linux runners to our Org private runners.

from internal, https://mobilecoin.slack.com/archives/CBD6RB4N4/p1649795053278909

but I could understand that we don't foresee this crate churning too much

---

GHA is free for public repos: https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration